inc/Base/Activate.php

```diff
@@ -56,13 +57,10 @@ private static function create_database() {
 				subdomain varchar(255) DEFAULT NULL,
 				username varchar(255) DEFAULT NULL,
 				password varchar(255) DEFAULT NULL,
-				autoresponder varchar(255) DEFAULT NULL,
```

Comment:
Please make an upgrade migration for existing installations. In Wordpress this can be achieved as described here - https://wordpress.stackexchange.com/a/240405

Reply:
Current version uses [DBdelta](https://developer.wordpress.org/reference/functions/dbdelta/) functionality. There is no need to migrate anything.

`The dbDelta function examines the current table structure, compares it to the desired table structure, and either adds or modifies the table as necessary, so it can be very handy for updates`
[Wordpress docs](https://codex.wordpress.org/Creating_Tables_with_Plugins)

Reply:
Awesome.